### DISCUSSION TITLE
delaunay_2d - Process finished with exit code -1073741571 (0xC00000FD)

### DISCUSSION THREAD

**Discussion by Jureq1234** [pyvista/pyvista#4899] [Q&A]
Hi,

I am using pyvista for my  research. When using delaunay 2d with a small number of points (1000) it works perfectly, but when I use 300000 points or more processes finished with exit code -1073741571 (0xC00000FD).

Where should I have look to fix the problem? The TIN surface is to complex? I have created it in Civil 3D with no problem.

Regards,
Jureq1234

**Comment by akaszynski** [comment]
The delaunay filter within VTK has issues with large meshes sometimes.  I'd try using a different algorithm (potentially [scipy.spatial.Delaunay.](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.Delaunay.html).
